src/FluentValidation/Internal/PropertyChain.cs

```diff
@@ -26,8 +26,8 @@ namespace FluentValidation.Internal;
 /// <summary>
 /// Represents a chain of properties
 /// </summary>
-public class PropertyChain {
-	readonly List<string> _memberNames = new(2);
+public class PropertyChain : List<string> {

```

Comment:
I'm not keen on this one - `PropertyChain` intentionally shouldn't expose all of the `List` operations. Can the changes to this file be reverted please? Unless your analysis indicates that this is a particularly critical one to keep, in which case if we can drop the `private List<string> _memberNames => this;` and replace all calls to `_memberNames` with `base` instead

Reply:
Resolved in 624bf1c667e935a62ddf0f6691d7375cd443da3c